projects/rocprofiler-systems/source/lib/core/trace_cache/metadata_registry.hpp

```diff
@@ -240,6 +242,10 @@ struct metadata_registry
     rocprofiler::sdk::callback_name_info_t<const char*> m_callback_tracing_info{
         rocprofiler::sdk::get_callback_tracing_names<const char*>()
     };
```

Comment:
```suggestion
    };

    using callback_rename_map_t =
        std::map<rocprofiler_tracing_operation_t, std::string_view>;

```
Consider defining a type for readability